src/Pages/CVsPage/api/useGetAllCvs.ts

```diff
@@ -0,0 +1,46 @@
+import {getCurrentUserID} from '@/App'
+import {gql, useQuery} from '@apollo/client'
+import {User} from 'cv-graphql'
+import {useEffect, useState} from 'react'
+import {toast} from 'react-toastify'
+
+export type IdArgs = {
+  userId: string
+}
+
+export type CvsResult = {
+  user: User
+}
+
+export const GET_CVS = gql`
+  query GetCvs($userId: ID!) {
+    user(userId: $userId) {
+      email
+      cvs {
+        id
+        name
+        education
+        description
+      }
+    }
+  }
+`
+
+export const useFetchCVs = () => {
+  const userId = getCurrentUserID()
+  const {data, loading, error, refetch} = useQuery<CvsResult, IdArgs>(GET_CVS, {
+    variables: {userId},
+    fetchPolicy: 'no-cache',
+  })
+  const [employee, setEmployee] = useState('')
+
+  useEffect(() => {
+    if (error) {
+      toast.error(error.message)
+    } else if (data) {
+      setEmployee(data.user.email)
+    }
+  }, [data, error])
```

Comment:
Выглядит, что как будто `employee` должен быть `string` или `null`. Также я думаю лучше переписать данный useEffect
```suggestion
  useEffect(() => {
    if (error) {
      toast.error(error.message)
    }
  }, [error])
  
  const employee = useMemo(() => !error && data ? data.user.email : null, [error, data]
```
или в return

```
{data, loading, error, refetch, employee: !error && data ? data.user.email : null}
```
Я бы рассмотрел второй вариант, если это будет работать